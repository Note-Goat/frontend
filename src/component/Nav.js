import {Link} from "react-router-dom";

export default function Nav() {
  return (
    <nav>
      <ul className="nav">
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/notebook">My Notebooks</Link>
        </li>
        <li>
          <Link to="/new-notebook">New Notebook</Link>
        </li>
        <li>
          <Link to="/account">My Account</Link>
        </li>
      </ul>
    </nav>
  );
}
