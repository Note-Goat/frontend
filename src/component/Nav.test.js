import renderer from 'react-test-renderer';
import Nav from "./Nav";
import {BrowserRouter} from "react-router-dom";

test('Nav component snapshot', () => {
  const tree = renderer
    .create(<BrowserRouter><Nav /></BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
