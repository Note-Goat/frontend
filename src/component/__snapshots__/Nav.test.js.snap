// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component snapshot 1`] = `
<nav>
  <ul
    className="nav"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/notebook"
        onClick={[Function]}
      >
        My Notebooks
      </a>
    </li>
    <li>
      <a
        href="/new-notebook"
        onClick={[Function]}
      >
        New Notebook
      </a>
    </li>
    <li>
      <a
        href="/account"
        onClick={[Function]}
      >
        My Account
      </a>
    </li>
  </ul>
</nav>
`;
